docs/install.md

```diff
@@ -162,6 +201,59 @@ Select the following tabs for the installation description of your target.
 
         Then exit `dietpi-config`. After this the first time installer procedure should run again from the start.
 
+    ??? important "Installation steps within the DietPi guest system when using the VirtualBox extension pack"
+        If you use the VirtualBox extension pack, after the DietPi base installation (done during the very first boot of the DietPi system), further installation steps within the DietPi guest system have to be done to achieve a working time synchronization. In general you have to use the following **Time sync mode** options set via the `dietpi-config` command (in the **Advanced Options**):
+
+        - VirtualBox without extension pack: The time synchronisation has to use options 1..4 ("Boot only", "Boot + Daily", "Boot + Hourly", "Daemon + Drift"). No further installation steps are necessary.
+        - VirtualBox with extension pack: The time synchronisation must use option 0 ("Custom"). Further installation steps are necessary, see below.
+
+            ![DietPi-config time synchronization](assets/images/dietpi-config-timesync.png){: width="600" height="328" loading="lazy"}
+
+        Additional installation steps in case of the use of the extension pack:
+
+        1. Check whether an optical disc with the VirtualBox extensions (`.iso`) is present:
+
+            ```sh
+            lsblk
+            ```
+
+            If you do not see a `/dev/sr0` optical drive, you need to add this in the virtual machine settings (in the shutdown state of the VM) `\Machine->Settings->Storage`: Add an optical storage which is linked to the guest additions (`.iso`).
+
+        2. Install the kernel headers and the extension pack:
+
+            ```sh
+            apt -y install build-essential dkms linux-headers-amd64
+            cd ~
+            mkdir /root/mnt
+            mount /dev/sr0 /root/mnt
+            cd /root/mnt
+            ./VBoxLinuxAdditions.run
+            ```
+
+            If you see an error message ***"VirtualBox Guest Additions: Kernel headers not found for target kernel"*** then the kernel headers are not installed properly (execute `apt -y install linux-headers-amd64` and afterwards try `./VBoxLinuxAdditions.run` again).
+
+        3. Reboot and check service  
+           Check a running VirtualBox extension pack service with
+
+             ```sh
+             systemctl status vboxadd-service
+             ```
+
+             The `vboxadd-service` should be in state *active*.
+
+        4. Settings in the host system  
+           The host system (not the DietPi guest system) also needs some settings that the time synchronization of the DietPi system runs fine. Execute:
+
+            ```sh
+            VBoxManage guestproperty set "<vm_name>" --timesync-set-start
+            VBoxManage guestproperty set "<vm_name>" "/VirtualBox/GuestAdd/VBoxService/--timesync-set-threshold" 1000
```

Comment:
`timesync-set-threshold` is the sync interval in seconds here? Crazy how many steps are required, at least on the Linux host, to make this, I'd say trivial and essential, task of host<>guest time sync working 😄.

On Windows it seems to be much easier, at in my case it worked OOTB (as far as I tested) after installing the guest additions on the guest system. Probably we should make it clear that this is for a Linux host only. Probably it wouldn't hurt on a Windows host either, but the `VBoxManage` command is not available by default on Windows cmd or PowerShell, i.e. it fails in most cases.

Reply:
@MichaIng
To be fool proof I omitted the `systemctl restart vboxadd-service`.  
Otherwise I need a complete full install test again. Maybe later...

The 20 minutes lead to the fact that after doing all these steps you need to wait a VM switched off time of 20 minutes to get all running (--timesync-set-threshold is not --timesync-interval).  
I took the value from the descriptions there: 
- <https://forums.virtualbox.org/viewtopic.php?f=6&t=39638>,
- <https://stackoverflow.com/questions/5308492/virtualbox-synchronisation-problems>, 
- <https://superuser.com/questions/463106/virtualbox-how-to-sync-host-and-guest-time>,
but I agree, 1 second is quite "dynamic". :-)  

So let it set to 10 seconds or discuss a bit more about this setting.

Reply:
As far as I understand, `--timesync-interval` is the general time sync interval, including drift, which defaults to 10 seconds already (other than the 5 minutes I remember we found somewhere else a while ago). `--timesync-set-threshold` then is the max time before the next timesync-interval won't only drift but fully sync the time, which defaults to 20 minutes. Setting this to 10 seconds as well, should then effectively disable drift and fully sync every 10 seconds instead. I didn't really understand what drift is about, if/why it is used as well by `systemd-timesyncd` by default. I mean getting an exact time via NTP and applying this as is to the system doesn't seem like a heavy task for me, compared to little adjustments that drift does? But otherwise the drift sync is what I remember makes `systemd-timesyncd` the faster/lighter solution, compared to the `ntp` daemon package. I'll read a bit into this topic, as it's interesting and good to know anyway.

Most importantly, with the other two settings, the time should be synced whenever the the VM starts. If it syncs every 10 seconds or only once an hour afterwards, should not matter much in most cases.